tests/test_quickbooks_start_date.py

```diff
@@ -9,7 +9,7 @@ class TestQuickbooksStartDate(TestQuickbooksBase):
     def expected_streams(self):
         """All streams are under test"""
         # Skipping stream deleted_objects due to data unavailability
-        return self.expected_check_streams().difference({'deleted_objects'})
+        return self.expected_check_streams().difference({'budgets', 'deleted_objects'})
```

Comment:
Can we incorporate CLIENT_JIRA here by creating a JIRA ticket for the data generation?
And update the comment as well like this:

Skipping stream budgets and deleted_objects due to data unavailability

Reply:
Added new test data and testing the budget stream so we can resolve this comment.